src/screens/docs/components/markdown.js

```diff
@@ -62,7 +62,8 @@ class Markdown extends React.Component {
     });
 
     md.use(markdownItTocAndAnchor, {
-      anchorLinkSymbol: "",
+      anchorLinkSymbol: "heading anchor link",
```

Comment:
Yeah you're right! My thought was to only describe the link but seeing the document outline like that it's pretty bad 😅.